test/js/data/symbol_bucket.test.js

```diff
@@ -72,5 +73,19 @@ test('SymbolBucket', function(t) {
     var b2 = collision.grid.keys.length;
     t.equal(a2, b2, 'detects collision and does not place feature');
 
+    // Use a custom console.warn to count warnings
```

Comment:
I would have to copy and paste [this](https://github.com/mapbox/mapbox-gl-js/blob/master/test/js/data/symbol_bucket.test.js#L23-L58) into a new test case unless you want me take the `bucketSetup()` function out of scope for the first test case?